lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -2486,9 +2486,10 @@ def main():
         supports_check_mode=True,
         add_file_common_args=True,
         required_if=[
+            ['state', 'present', ['provider']],
             ['provider', 'entrust', ['entrust_requester_email', 'entrust_requester_name', 'entrust_requester_phone',
                                      'entrust_api_user', 'entrust_api_key', 'entrust_api_client_cert_path',
-                                     'entrust_api_client_cert_key_path']]
+                                     'entrust_api_client_cert_key_path']],
```

Comment:
I don't think the extra ',' here would hurt, are there any relevant style guidelines?

Reply:
I don't think there's anything on that.

I prefer trailing commas in general, since they reduce diff size when adding new entries later-on. If there's no trailing comma, this line needs to be modified as well.